### PR TITLE
[add] Menu animations

### LINKED ISSUE #56: Use CSSTransitionGroup instead of CSS animation for menus
Right now menus fade in but immediately snap out. We could fix this using the CSSTransitionGroup addon.

### PR DESCRIPTION
**Pull request by nhunzaker**
Fixes #56 

cc @hellokv 

menu

### CODE REVIEW

File: style/components/menu.scss [hunk 1]
```diff
@@ -81,7 +82,32 @@ $col-menu-z-index        : 1 !default;
...
-    opacity: 0.25;
+    color: rgba(#000, 0.2);
     pointer-events: none;
   }
 }
+
+.col-menu-enter,
+.col-menu-leave.col-menu-leave-active {
+  opacity: 0;
+  transform: translateZ(0) scale(0.2);
```
Comment (averyvery) [inline style/components/menu.scss]: Are there still visual glitches that this prevents?

**Comment by averyvery** [comment]
:+1:

**Comment by solomonhawk** [comment]
:+1:

File: style/components/menu.scss (hunk 1 above)
Comment (nhunzaker) [inline style/components/menu.scss]: Good call, no! What I thought was a layering issue was actually z-index.

Comment (nhunzaker) [inline style/components/menu.scss]: So it was a layering issue, just not the one I expected.

**Pull request merged**